Tribler/Core/Session.py

```diff
@@ -648,6 +651,41 @@ def create_channel(self, name, description, mode=u'closed'):
         """
         return self.lm.channel_manager.create_channel(name, description, mode)
 
+    @blocking_call_on_reactor_thread
```

Comment:
I haven't looked at it, but if this is being called from the events thingie, it will be called on the threadpool